deployments/roman/image/environments/frozen/mirage-frozen.yml

```diff
@@ -0,0 +1,366 @@
+name: mirage
```

Comment:
Because Roman was having trouble building on GitHub,  and because it was just a data-less paperweight,   I dropped the Mirage kernel from Roman.   So in theory this should not be here any more.   IDK if it will cause trouble or not.